projects/ti_to_ti/README.md

```diff
@@ -76,6 +88,133 @@ and set up the correct model path and dataset path, then run:
 bash scripts/sft_ti_to_ti.sh
```

Comment:
It is vague here. So the `model_name_or_path` here should be the `pt` file which is after the pre-tokenization ?

Reply:
Fixed in https://github.com/PKU-Alignment/align-anything/pull/46/commits/9b65944e86268263ef27c38e063a06a264cdf74b, I claimed in all the scripts what the arguments should look like